### ISSUE TITLE
Confusing about inferencing the single pointcloud

### ISSUE THREAD

**Issue by ThomasVercetti** [lulutang0608/Point-BERT#40]
@yuxumin  @lulutang0608  Thanks for sharing the paper and code.
One point I'm confused about is whether I can use the model to input a single point cloud and get the lower-dimension representation.

**Comment by yuxumin** [comment]
yes, inference the point cloud with the pretrained dVAE, and represent the point cloud by a list of int with `argmax(logits)` https://github.com/lulutang0608/Point-BERT/blob/master/models/dvae.py#L335
